packages/design-system/src/components/web-components/ds-autocomplete/ds-autocomplete.stories.tsx

```diff
@@ -140,9 +140,24 @@ const Template = (args) => {
 
   let filteredItems = null;
   if (input.length > 0 && items) {
-    filteredItems = items.filter(
-      (item) => !item.name || item.name.toLowerCase().includes(input.toLowerCase())
-    );
+    filteredItems = items
+      .map((item) => {
+        if (item.label && item.items) {
+          // Handle grouped items
+          const filteredGroupItems = item.items.filter(
+            (groupItem) =>
+              !groupItem.name || groupItem.name.toLowerCase().includes(input.toLowerCase())
+          );
+          return {
+            ...item,
+            items: filteredGroupItems,
+          };
```

Comment:
Is the intention for a group label to always be visible in the list even if it has no matching items?

Reply:
Good point! I agree—it would make sense to filter out empty groups.